src/bp/core/routers/auth.ts

```diff
@@ -192,6 +196,19 @@ export class AuthRouter extends CustomRouter {
         res.sendStatus(200)
       })
     )
+
+    // Temporary route to obtain a token when using cookie authentication, for the bp pull/push command
+    router.get(
+      '/getToken',
```

Comment:
I'm not a big fan of doubling the get word but since thus is temporary it's fine